fhir_extractor/src/main.py

```diff
@@ -39,13 +37,21 @@ def process_event(msg):
     try:
         logger.debug("Getting Mapping")
         resource_mapping = get_resource_from_id(resource_id=resource_id)
+
+        # Get credentials
+        if not resource_mapping["source"]["credential"]:
+            raise MissingInformationError("credential is required to run fhir-river by batch.")
+
+        credentials = resource_mapping["source"]["credential"]
+        extractor.update_connection(credentials)
+
         analysis = analyzer.analyze(resource_mapping)
         run_resource(resource_mapping, analysis, primary_key_values, batch_id)
 
-        return "Success", 200
+    except Exception as err:
+        logger.error(err)
```

Comment:
Do not return Success, 200 if there is an error

Reply:
Oops, nice catch 👍

Reply:
I would return the Error with status code 400, or raise an exception, not just log

Reply:
How does this work? How is the return of this function processed by kafka?

Reply:
Ohlala sorry, I thought it was the API function...
So actually, there is no need to return anything in this case, event the `Return "Success, 200" can be removed. I think we forgot to remove it in a previous PR

Reply:
I'll remove it then 🙂